packages/@expo/cli/src/run/ios/appleDevice/client/UsbmuxdClient.ts

```diff
@@ -18,21 +18,26 @@ const debug = Debug('expo:apple-device:client:usbmuxd');
 
 export interface UsbmuxdDeviceProperties {
   /** 480000000 */
-  ConnectionSpeed: number;
+  ConnectionSpeed?: number;
   /** 'USB' */
-  ConnectionType: 'USB';
+  ConnectionType: 'USB' | 'Network';
   /** 7 */
   DeviceID: number;
   /** 339738624 */
-  LocationID: number;
+  LocationID?: number;
   /** 4776 */
-  ProductID: number;
+  ProductID?: number;
   /** '00008101-001964A22629003A' */
   SerialNumber: string;
   /** '00008101-001964A22629003A' */
-  UDID: string;
+  UDID?: string;
```

Comment:
Should document when the ID is not defined since we basically use it for everything related to identifying the device.